package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-prettier": "4.0.0",
+    "execa": "^5.0.0",
```

Comment:
This dependency actually came with `Jest` already, hence no changes in `yarn.lock`. Also note that version 6.x doesn't work here, because of the same reason that `require`ing it doesn't work then.